lib/practice.md

```diff
@@ -3,58 +3,74 @@
 - What package did you install for enumeration?
 
 ```md
-<!-- your answer here -->
+lodash
 ```
 
 ## Research: Popular NPM Packages
 
 Package 1:
 ```md
-<!-- your answer here -->
+async
 ```
 1.  What problem does this package solve?
 ```md
-<!-- your answer here -->
+asynchronous control flow
 ```
 1.  How can I use this package in my own code? What steps are involved?
 ```md
-<!-- your answer here -->
+`npm install --save async`
+
+require async in the file where I want to use it
+
+cry myself to sleep because I hate this library so much
 ```
 1.  How well-maintained is this package? What criteria did I use to decide?
 ```md
-<!-- your answer here -->
+seems pretty well maintained, based on frequency of commits and number of active and closed issues/pull requests.
 ```
 
 Package 2:
 ```md
-<!-- your answer here -->
+underscore
 ```
 1.  What problem does this package solve?
 ```md
-<!-- your answer here -->
+I honestly do not know what problem this solves. Probably JavaScript devs not having enough random dependencies.
```

Comment:
It adds some nice utility functions that JavaScript doesn't have built-in, such as deep equality for objects.